File: 240116/연속되는 수 2/continuous-number2.cpp
#include <iostream>
#include <algorithm>
#include <climits>
using namespace std;
#define MAX_R 1000

int n;
int arr[MAX_R];

int main() {
    // 여기에 코드를 작성해주세요.
    cin >> n;

    for(int i=0; i<n; i++) {
        cin >> arr[i];
    }

    int max_cnt = INT_MIN;
    int cnt = 0;
    for(int i=0; i<MAX_R; i++) {
        if(i ==0 || arr[i] == arr[i-1]) cnt++;
        else {
            max_cnt = max(max_cnt, cnt);
            cnt = 1;
        }
    }
    cout << max_cnt;

    return 0;
}

File: 240820/연속되는 수 2/continuous-number2.py
n = int(input())

nums = [
    int(input())
    for _ in range(n)
]

max_cnt, cnt = 0, 0

for i in range(n):
    if(i==0 or nums[i-1] != nums[i]):
        cnt = 1
    else:
        cnt += 1

    max_cnt = max(max_cnt, cnt)


print(max_cnt)